exec/java-exec/src/main/java/org/apache/drill/exec/physical/impl/join/HashJoinBatch.java

```diff
@@ -1212,6 +1238,81 @@ private void updateStats() {
     this.stats.setLongStat(Metric.SPILLED_PARTITIONS, numSpilled);
   }
 
+  private void drainLeft() {
```

Comment:
Follow on (after #1480) - these two methods can be eliminated ( `drainLeft()` and `drainRight()` )